I can't build the yami-apache with libva2.00

vaapipostprocess_scaler.cpp: In member function ‘virtual YamiStatus YamiMediaCodec::VaapiPostProcessScaler::process(const std::shared_ptr<VideoFrame>&, const std::shared_ptr<VideoFrame>&)’:
vaapipostprocess_scaler.cpp:129:15: error: ‘VAProcPipelineParameterBuffer {aka struct _VAProcPipelineParameterBuffer}’ has no member named ‘rotation_state’
     vppParam->rotation_state = mapToVARotationState(m_transform);
               ^~~~~~~~~~~~~~
make[2]: *** [Makefile:706：libyami_vpp_la-vaapipostprocess_scaler.lo] 错误 1


I think it may be wrong of the version of libva ?
Please make sure it can be build in all of version of libva?!

--- a/vpp/vaapipostprocess_scaler.cpp
+++ b/vpp/vaapipostprocess_scaler.cpp
@@ -126,10 +126,13 @@ VaapiPostProcessScaler::process(const SharedPtr<VideoFrame>& src,
         vppParam->filters = &filters[0];
         vppParam->num_filters = (unsigned int)filters.size();
     }
+#if VA_CHECK_VERSION(1, 1, 0)
     vppParam->rotation_state = mapToVARotationState(m_transform);
+#endif
 
     return picture.process() ? YAMI_SUCCESS : YAMI_FAIL;
 }
+
 uint32_t VaapiPostProcessScaler::mapToVARotationState(VppTransform vppTransform)
 {
     switch (vppTransform) {
@@ -410,13 +413,16 @@ VaapiPostProcessScaler::setParameters(VppParamType type, void* vppParam)
         return setColorBalanceParam(*colorbalance);
     }
     else if (type == VppParamTypeTransform) {
+#if VA_CHECK_VERSION(1, 1, 0)
         VppParamTransform* param = (VppParamTransform*)vppParam;
         if (param->size != sizeof(VppParamTransform)) {
             return YAMI_INVALID_PARAM;
         }
         m_transform = (VppTransform)param->transform;
-
         return YAMI_SUCCESS;
+#else
+        return YAMI_INVALID_PARAM;
+#endif
     }
     return VaapiPostProcessBase::setParameters(type, vppParam);
 }

